defitrack-protocols/Maple/src/main/java/io/defitrack/protocol/Maple/MapleV2MainnetService.kt

```diff
@@ -0,0 +1,74 @@
+package io.defitrack.protocol.maple
+
+import com.fasterxml.jackson.core.type.TypeReference
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.defitrack.protocol.maple.domain.MapleReserve
+import io.defitrack.protocol.maple.domain.UserReserve
+import io.defitrack.thegraph.TheGraphGatewayProvider
+import org.springframework.stereotype.Component
+
+@Component
+class MapleV2MainnetService(
+    graphGatewayProvider: TheGraphGatewayProvider,
+    private val objectMapper: ObjectMapper,
+) {
+
+    val thegraph =
+        graphGatewayProvider.createTheGraphGateway("https://api.thegraph.com/subgraphs/name/maple/protocol-v3")
```

Comment:
can't find this graph

Reply:
I'm also pretty sure maple doesn't have the same structure as Aave, so not sure what the contracts you copied from the Aave protocol are referring to.

Reply:
I changed everything and created fresh folders with fresh files after a bit of research into what's needed here